state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -2181,11 +2171,12 @@ pub(crate) mod utilities {
 
 	pub(super) fn calculate_swap_refund_parameters(
 		params: &ChannelRefundParameters,
-		deposit_block: u32,
+		execute_at_block: u32,
 		input_amount: AssetAmount,
 	) -> SwapRefundParameters {
 		SwapRefundParameters {
-			refund_block: deposit_block.saturating_add(params.retry_duration),
+			// NOTE: We add 1 to make retry duration inclusive of the last block:
+			refund_block: execute_at_block.saturating_add(params.retry_duration).saturating_add(1),
```

Comment:
I don't understand why this was added - is it necessary/signficant?

Reply:
You are right, it wasn't necessary, reverted. I think I forgot that we still execute one last time at `refund_block` before refunding as that block, so it already works as intended.